rust/examples/mobile-client.rs

```diff
@@ -103,7 +103,7 @@ fn main() -> Result<(), ()> {
         .init();
 
     // create a new client
-    let client = MobileClient::init(&opt.url, get_participant_settings());
+    let client = MobileClient::init(&opt.url, get_participant_settings()).unwrap();
```

Comment:
It would be nice to add a comment explaining why unwrapping is OK here and in the other instances of this.

Reply:
Actually, we should not unwrap here. The init can fail as far as I can see. Shouldn't we exit gracefully with an error message?

Reply:
This file is an example that I only use to test the mobile client implementation. My plan is to remove it once we moved the sdk into the repository.

Reply:
Oh sorry I missed that.